### PR TITLE
Abap pragma

### PR DESCRIPTION
**Pull request by christianguenter2**
#854

### CODE REVIEW

File: src/zabapgit_object_prag.prog.abap [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+
+  METHOD lif_object~has_changed_since.
+
+    rv_changed = abap_true.
+
+  ENDMETHOD.
+
+  METHOD lif_object~changed_by.
+
+    rv_user = sy-uname.
```
Comment (larshp) [inline src/zabapgit_object_prag.prog.abap]: change to `c_user_unknown`

File: src/zabapgit_object_prag.prog.abap [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+
+  ENDMETHOD.
+
+  METHOD lif_object~serialize.
+
+    DATA: lo_pragma TYPE REF TO cl_abap_pragma,
+          pragma    TYPE lcl_object_prag=>ty_pragma.
+
+    TRY.
+        lo_pragma = cl_abap_pragma=>get_ref( ms_item-obj_name ).
```
Comment (larshp) [inline src/zabapgit_object_prag.prog.abap]: do you know if this works on 702? as far as I remember pragmas was introduced sometime later

Comment (christianguenter2) [inline src/zabapgit_object_prag.prog.abap]: Afaik they were introduced with 702. But I don't have a system to check.

Comment (christianguenter2) [inline src/zabapgit_object_prag.prog.abap]: http://zevolving.com/2013/02/pragmas-in-abap-replacing-pseudo-comments/

Comment (larshp) [inline src/zabapgit_object_prag.prog.abap]: cool 👍

**Pull request merged**